sources/osgUtil/NodeGizmo.js

```diff
@@ -972,19 +994,24 @@ utils.createPrototypeNode(
 
             var v = vec2.create();
             getCanvasCoord(v, e);
-            if (vec2.distance(v, this._downCanvasCoord) === 0.0) this.pickAndSelect(e);
+            if (vec2.distance(v, this._downCanvasCoord) <= this._pixelsMargin) this.pickAndSelect(e);
 
             this._showAngle.setNodeMask(0x0);
             this._isEditing = false;
             if (!this._hoverNode) return;
             this.updateGizmoMask();
+
+            // to deselect the _hoverNode in touch devices
+            if (('ontouchstart' in window || navigator.maxTouchPoints || navigator.msMaxTouchPoints))
+                this.onMouseMove();
         },
 
         onMouseMove: function(e) {
             if (!this._attachedNode) return;
             var hit;
             if (this._isEditing === false) {
-                hit = this.pickGizmo(e, NodeGizmo.PICK_GIZMO);
+                if (e)
+                    hit = this.pickGizmo(e, NodeGizmo.PICK_GIZMO);
                 this.onNodeHovered(hit);
```

Comment:
'hit' will be 'undefined' if 'e' is 'undefined'. Is the method onNodeHovered ignoring correctly a undefined hit?

Reply:
Yes (line 415)